propka/lib.py

```diff
@@ -162,15 +167,13 @@ def loadOptions(*args):
     parser.add_option("-v", "--version", dest="version_label", default="Jan15",
            help="specifying the sub-version of propka [Jan15/Dec19]")
     parser.add_option("-p", "--parameters",dest="parameters", default=pkg_resources.resource_filename(__name__, "propka.cfg"),
-                      help="set the parameter file [%default]")
-    parser.add_option("-z", "--verbose", dest="verbose", action="store_true", default=True,
-           help="sleep during calculations")
-    parser.add_option("-q", "--quiet", dest="verbose", action="store_false",
-           help="sleep during calculations")
-    parser.add_option("-s", "--silent",  dest="verbose", action="store_false",
-           help="not activated yet")
-    parser.add_option("--verbosity",  dest="verbosity", action="store_const",
-           help="level of printout - not activated yet")
+           help="set the parameter file [%default]")
+    parser.add_option("--verbosity", dest="verbosity", action="store_const", default=1,
```

Comment:
I think we should remove the --verbosity option (it wasn't working previously anyway), since we already have -q and -z options that do the same thing (-q suppresses all logging - verbosity=0, and -z enables debug mode - verbosity=2).